extensions/windows-event-log/ConsumeWindowsEventLog.cpp

```diff
@@ -635,6 +653,8 @@ void ConsumeWindowsEventLog::putEventRenderFlowFileToSession(const EventRender&
       }
     }
     session.putAttribute(flowFile, FlowAttributeKey(MIME_TYPE), "application/xml");
+    session.putAttribute(flowFile, "Timezone name", wel::to_string(tzstr.c_str()));
+    session.putAttribute(flowFile, "Timezone bias", tzbias);
```

Comment:
I would call this "Timezone offset".  I think "bias" and "daylight bias" are Microsoft coinages for the two components of the offset, but here we are printing their sum.

Reply:
Done